enhancements/bpfd-on-ocp.md

```diff
@@ -0,0 +1,418 @@
+---
+title: bpfd-on-ocp
+authors:
+  - "@anfredette"
+  - "@astoycos"
+  - "@Billy99"
+  - "@dave-tucker"
+reviewers:
+  - "@TBD"
```

Comment:
We will want names from the relevant teams here.

Reply:
Done.